src/radio-widget/containers/radio-station-list/radio-station-widget.js

```diff
@@ -27,14 +25,14 @@ const RadioStationWidget = ({currentlyPlayingIndex}) => {
             </div>
             <RadioStationList 
                 onRadioSelect={onRadioSelect}
-                currentlyPlayingIndex={state.currentlyPlayingIndex}
+                currentlyPlayingIndex={currentlyActiveIndex}
                 stations={stationsList}
             />
             <div class="radioListFooter">
-                <div className={stationsList[state.currentlyPlayingIndex] ? "footer-visible" : "footer-hidden"}>
+                <div class="footer-visible">
                     <div class="footer">Currently playing</div>
                     <div class="currently-playing-title">
-                        {stationsList[state.currentlyPlayingIndex]?.title}
+                        {stationsList[currentlyActiveIndex]?.title}
```

Comment:
you can create `const currentStation = stationsList[currentlyActiveIndex]; ` after the hooks